packages/block-editor/src/components/spacing-sizes-control/input-controls/spacing-input-control.js

```diff
@@ -222,13 +224,26 @@ export default function SpacingInputControl( {
 						}
 						value={ currentValue }
 						units={ units }
-						min={ minimumCustomValue }
+						min={ minValue }
 						placeholder={ allPlaceholder }
 						disableUnits={ isMixed }
 						label={ ariaLabel }
 						hideLabelFromVision
 						className="spacing-sizes-control__custom-value-input"
 						size={ '__unstable-large' }
+						onDragStart={ () => {
+							if ( value?.charAt( 0 ) === '-' ) {
+								setMinValue( 0 );
```

Comment:
Would this lower bound still make sense if `minimumCustomValue` was > 0? `Math.max( 0, minimumCustomValue )` could be more robust in the general case.